src/documentation/ideas/IdeasDocumentation.tsx

```diff
@@ -92,6 +92,7 @@ const ActiveLevel = ({
             mt={1}
             mb={1}
             sx={{
+              "& ol": { pl: 4 },
```

Comment:
We have this in ReferenceTopicEntry (from 102)

```
          listStylePos="inside"
          className="docs-code"
          sx={{
            "& ul": { listStyleType: "disc", pl: 3 },
            ...docStyles,
          }}
```

Perhaps the list style pos, ul and ol styling should all move to docStyles?

Then make the padding consistent one way or another.

Reply:
Changes made: [`879c77b`](https://github.com/microbit-foundation/python-editor-next/pull/833/commits/879c77bfb65da4653c60916cb9dd9f45143b57b8), [`e74bc57`](https://github.com/microbit-foundation/python-editor-next/pull/833/commits/e74bc57aa9c8e683a27990d4a6f04f514e3e5daa).